src/applications/vaos/manifest.json

```diff
@@ -0,0 +1,8 @@
+{
+  "appName": "VA Online Scheduling",
+  "entryFile": "./vaos-entry",
+  "entryName": "vaos-beta",
+  "rootUrl": "/health-care/vaos-beta",
+  "noRouting": true,
```

Comment:
Looks like `noRouting` doesn't really do anything anymore, plus this app does have routing, so I think we can remove it.